undebt/cmd/main.py

```diff
@@ -57,21 +57,21 @@ def _write_result_text(result_text, path, dry_run):
 def _handle_arguments():
     parser = argparse.ArgumentParser(prog='undebt')
     parser.add_argument(
-        '--input', '-i', metavar='path', action='append',
```

Comment:
I thought #27 mentioned that we want to make this break change? otherwise we don't need this PR.